docs/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
```

Comment:
This isn't really true though ... think of the plugin .rb files for stuff like changetracking, security, etc.

Reply:
At least someone should be involved in every PR

Reply:
This way we can track any change made to OMS repo